lib/index.js

```diff
@@ -280,8 +280,8 @@ function _checkTalkNamespace(title, siteInfo) {
 }
 
 function _checkMaxLength(title, namespace) {
-    const maxLength = namespace.isSpecial() ? 512 : 256;
-    if (title.length > maxLength) {
+    const maxLength = namespace.isSpecial() ? 512 : 255;
+    if (Buffer.from(title).length > maxLength) {
```

Comment:
Better to use [Buffer.byteLength](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_bytelength_string_encoding)

Reply:
Replaced with byteLength